test/sharness/lib/test-lib.sh

```diff
@@ -287,7 +287,7 @@ test_launch_ipfs_daemon_without_network() {
 
 do_umount() {
   if [ "$(uname -s)" = "Linux" ]; then
-  fusermount -u "$1"
+  fusermount -z -u "$1"
```

Comment:
this is "lazy unmount" so we don't have to sleep after unmounting so that we don't get errors when removing mount dirs too quickly